website/docs/docs/explore/access-dbt-insights.md

```diff
@@ -7,7 +7,7 @@ tags: [dbt Insights]
 image: /img/docs/dbt-insights/insights-chart.jpg
 ---
```

Comment:
@matthewshaver and @dave-connors-3 make sure to change all of the "unlisted: true" to false

Reply:
Will take this action on the internal repo for the 28th